src/pod-security-policy/seccomp/src.rego

```diff
@@ -99,7 +70,7 @@ get_allowed_profiles[allowed] {
 # The simply translated profiles
 get_allowed_profiles[allowed] {
     profile := input.parameters.allowedProfiles[_]
-    not startswith(lower(profile), "localhost")
```

Comment:
Instead of keeping `naming_translation` we can:

* if there is a "/" in the name, use the name directly (assume its already in annotation format)
* If there is no "/" in the name, build a profile object (via a function call), then call canonicalize_seccomp_profile

This ensures we use the same translation logic everywhere.